app/src/main/java/com/example/android/guesstheword/screens/game/GameViewModel.kt

```diff
@@ -0,0 +1,83 @@
+package com.example.android.guesstheword.screens.game
+
+import android.util.Log
+import androidx.lifecycle.ViewModel
+
+class GameViewModel : ViewModel() {
+
+    // The current word
+    var word = ""
+
+    // The current score
+    var score = 0
+
+    // The list of words - the front of the list is the next word to guess
+    private lateinit var wordList: MutableList<String>
+
+    init {
+        Log.i("GameViewModel", "GameViewModel created!")
```

Comment:
Please delete Logs

Reply:
In the next pull request I will delete the logs